scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -56,11 +57,34 @@ workflow case_gatk_acnv_workflow {
         mem=2
   }
 
-  call NormalizeSomaticReadCounts as TumorNormalizeSomaticReadCounts {
+  call AnnotateTargets as TumorAnnotateTargets {
```

Comment:
Since the tumor and normal samples share targets, there should only be a single `AnnotateTargets` task.